packages/experimental/src/inbox-note/style.scss

```diff
@@ -30,7 +30,8 @@
 	}
 
 	&:not(.message-is-unread) {
-		h3 {
+		h3,
+		h4 {
```

Comment:
Is there a particular reason for adding these styles for `h4` as well?

Reply:
I was just looking at this and wondering about it too. Could a class selector like `.woocommerce-inbox-message__title` be used instead, or are there specificity issues doing that?

Reply:
Thank you @louwie17 and @adrianduffell. That's a good question. I should have left an explanation.

I noticed note titles are rendered with H4 in the activity panel.
![Screen Shot 2021-12-02 at 11 35 36 AM](https://user-images.githubusercontent.com/4723145/144491620-fe460db3-b3df-414f-a05e-db4f412f932f.jpg)

@adrianduffell That's a good suggestion :) Updatd in https://github.com/woocommerce/woocommerce-admin/pull/7983/commits/f6b758e3b5d21e27bb3b6fb23423772a7ac9e62b